book-worm-2010/book-worm-fe
# Feature/implement rainbow path

### What does this PR do? 
>Clearly and concisely describe the feature. 

Implements the data displayed for the user's reading progress with book title displayed on the x-axis, worm avatar, and waves paths for full visualization.  

### Is this a feature or Fix/Refactor?  

Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

Initially, the plan was to display book images for each book in progress but there were some issues getting the images to format correctly.  For now, the titles still satisfy MVP and we will implement the images later on.  

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

Home.js, CurrentBookRainbow.js and CurrentBookRainbow.scss and corresponding image files.

### How should this be tested?
>What steps need to be taken to confirm functionality? 

Run `npm start` in terminal to open the application in your browser on `localhost:3000` to view the data visualization in progress with dummy data for now.  

### Steps To Take
>Any questions and possible steps moving forward.
Connect to API and post books to the user's book list to fully test functionality and whether the progress is being displayed accurately when the bookmark feature is implemented.
